File: backend/MoneyManagerBackend/TransactionLoader/Transaction.cs
using CsvHelper.Configuration.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransactionLoader
{
	public class Transaction
	{
		[Name("Account Number")]
		public string AccountNumber { get; set; }

		[Name("Transaction Date")]
		public DateTime Date { get; set; }

		[Name("Description 1")]
		public string Description1 { get; set; }

		[Name("Description 2")]
		public string Description2 { get; set; }

		[Name("CAD$")]
		public double AmountCad { get; set; }
	}
}

File: frontend/money-manager-front/src/components/Transaction.js

const Movement = ({movement}) => {
    return (
        <div >
            <p>Date: {movement.date}</p>
            <p>Amount: {movement.amount}</p>
            <p>Description: {movement.description}</p>
            <p>Account: {movement.account}</p>
        </div>
    )
}

export default Movement
